package.json

```diff
@@ -3,34 +3,78 @@
+    "e2e": "yarn cypress install && yarn grafana-e2e run",
+    "e2e:update": "yarn cypress install && yarn grafana-e2e run --update-screenshots",
+    "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
+    "lint:fix": "yarn lint --fix",
+    "server": "docker-compose up --build",
+    "test": "jest --watch --onlyChanged",
+    "test:ci": "jest --maxWorkers 4",
     "test:coverage": "jest --coverage",
     "test:coverage:changes": "jest --coverage --changedSince=origin/main",
-    "dev": "grafana-toolkit plugin:dev",
-    "watch": "grafana-toolkit plugin:dev --watch"
+    "typecheck": "tsc --noEmit"
   },
   "author": "Grafana",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@babel/core": "^7.16.7",
     "@grafana/aws-sdk": "0.0.37",
-    "@grafana/data": "8.1.5",
-    "@grafana/runtime": "8.1.5",
-    "@grafana/toolkit": "8.1.5",
-    "@grafana/ui": "8.1.5",
-    "@testing-library/react": "^10.4.4",
+    "@grafana/e2e": "9.1.2",
+    "@grafana/e2e-selectors": "9.1.2",
+    "@grafana/eslint-config": "^2.5.0",
+    "@grafana/tsconfig": "^1.2.0-rc1",
+    "@swc/core": "^1.2.144",
+    "@swc/helpers": "^0.3.6",
+    "@swc/jest": "^0.2.20",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.3",
+    "@types/glob": "^8.0.0",
+    "@types/jest": "^27.4.1",
     "@types/lodash": "latest",
+    "@types/node": "^17.0.19",
     "@types/prismjs": "1.16.0",
     "@types/slate": "0.47.1",
-    "jest-environment-jsdom-sixteen": "^1.0.3"
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
+    "copy-webpack-plugin": "^10.0.0",
+    "css-loader": "6.7.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-jsdoc": "^37.7.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.26.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-webpack-plugin": "^3.1.1",
+    "fork-ts-checker-webpack-plugin": "^7.2.0",
+    "glob": "^8.0.3",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "27.5.0",
+    "jest-environment-jsdom-sixteen": "^1.0.3",
+    "prettier": "^2.5.0",
+    "replace-in-file-webpack-plugin": "^1.0.6",
+    "sass": "1.55.0",
+    "sass-loader": "13.1.0",
+    "style-loader": "3.3.1",
+    "swc-loader": "^0.1.15",
+    "ts-node": "^10.5.0",
+    "tsconfig-paths": "^3.12.0",
+    "typescript": "^4.4.0",
+    "webpack": "^5.69.1",
+    "webpack-cli": "^4.9.2",
+    "webpack-livereload-plugin": "^3.0.2"
   },
   "dependencies": {
-    "@emotion/css": "11.1.3",
+    "@emotion/css": "^11.1.3",
+    "@grafana/data": "8.1.5",
+    "@grafana/runtime": "8.1.5",
+    "@grafana/ui": "8.1.5",
     "prismjs": "1.27.0",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
     "react-use": "^15.3.4",
     "rxjs": "6.6.3",
-    "ts-loader": "6.2.1",
-    "tslib": "2.2.0",
-    "typescript": "4.3.4"
+    "tslib": "2.2.0"
```

Comment:
Don't think any of the packages that are specified as external deps [here](https://github.com/grafana/x-ray-datasource/blob/881ce9164852a397de5e8ba8786b748164f8f4e1/.config/webpack/webpack.config.ts#L32) should be ´dependencies´ in package.json. Can you make them devDependencies instead and see if it still works? 

`externals` means that the version of these packages should come from core grafana and not the plugin. That is because we don't want core grafana and the plugin to run different versions of for example the ui.

Reply:
done!